_topic_maps/_topic_map_osd.yml

```diff
@@ -1178,3 +1178,285 @@ Topics:
   Topics:
   - Name: Serverless overview
     File: about-serverless
+---
+Name: Virtualization
+Dir: virt
+Distros: openshift-dedicated
+Topics:
+- Name: About
+  Dir: about_virt
+  Topics:
+  - Name: About OpenShift Virtualization
+    File: about-virt
+    Distros: openshift-dedicated
+  - Name: About OKD Virtualization
+    File: about-virt
+    Distros: openshift-origin
+  - Name: Security policies
+    File: virt-security-policies
+  - Name: Architecture
+    File: virt-architecture
+    Distros: openshift-dedicated
+- Name: Release notes
+  Dir: release_notes
+  Topics:
+  - Name: OpenShift Virtualization release notes
+    File: virt-release-notes-placeholder
+    Distros: openshift-dedicated
+  # - Name: OKD Virtualization release notes
+  #   File: virt-release-notes-placeholder
+  #   Distros: openshift-origin
+- Name: Getting started
+  Dir: getting_started
+  Topics:
+  - Name: Getting started with OpenShift Virtualization
+    File: virt-getting-started
+    Distros: openshift-dedicated
+  - Name: Getting started with OKD Virtualization
+    File: virt-getting-started
+    Distros: openshift-origin
+  - Name: virtctl and libguestfs
+    File: virt-using-the-cli-tools
+  - Name: Web console overview
+    File: virt-web-console-overview
+    Distros: openshift-dedicated
+- Name: Installing
+  Dir: install
+  Topics:
+  - Name: Preparing your cluster
+    File: preparing-cluster-for-virt
+  - Name: Installing OpenShift Virtualization
+    File: installing-virt
+  - Name: Uninstalling OpenShift Virtualization
+    File: uninstalling-virt
+- Name: Post-installation configuration
+  Dir: post_installation_configuration
+  Topics:
+  - Name: Post-installation configuration
+    File: virt-post-install-config
+  - Name: Node placement rules
+    File: virt-node-placement-virt-components
+  - Name: Network configuration
+    File: virt-post-install-network-config
+  - Name: Storage configuration
+    File: virt-post-install-storage-config
+- Name: Updating
+  Dir: updating
+  Topics:
+  - Name: Updating OpenShift Virtualization
+    File: upgrading-virt
+    Distros: openshift-dedicated
+  - Name: Updating OKD Virtualization
+    File: upgrading-virt
+    Distros: openshift-origin
+- Name: Virtual machines
+  Dir: virtual_machines
+  Topics:
+  - Name: Creating VMs from Red Hat images
+    Dir: creating_vms_rh
+    Topics:
+    - Name: Creating VMs from Red Hat images overview
+      File: virt-creating-vms-from-rh-images-overview
+    - Name: Creating VMs from templates
+      File: virt-creating-vms-from-templates
+# Hiding, as feature is TP as of 4.14
```

Comment:
Are these docs for 4.14 or 4.15 ?
in 4.15, VM instance types are no longer in tech preview

Reply:
While ROSA is versionless, OpenShift Virtualization is a self managed operator, so I guess users (the customer admins) will know the OpenShift Virtualization version they are using.

Reply:
@orenc1 @ronensdeor Would a note in ROSA/OSD docs make sense here:

> OpenShift Virtualization 4.15 and higher is supported for use on Red Hat OpenShift Service on AWS clusters. Because OpenShift Virtualization 4.14 is a Technology Preview feature, is not supported for use on Red Hat OpenShift Service on AWS clusters

Reply:
Hi,
OpenShift Virtualization is GA on ROSA since 4.14.
The Tech Preview feature that Oren referred to is how to create VMs in OpenShift Virtualization: In OpenShift Virtualization 4.14 instance types were a tech preview and in 4.15 they are GA.